po/fr.po

```diff
@@ -8765,7 +8765,7 @@ msgstr "id lorsque chargé"
 
 #: ../src/gui/accelerators.c:1926
 msgid "clear device first"
-msgstr "effacer la liste des périphériques"
+msgstr "nettoyer d'abord le périphérique"
```

Comment:
This one read strange to me. "nettoyer" n'est probablement pas la bonne traduction pour clear qui a plusieurs sens en Anglais. déblayer, nette, clair, débarrasser, libérer... Je n'ai pas regardé le contexte cependant.

Reply:
The meaning is: "clear the device's shortcuts list before importing" (then, importing an empty list is equivalent of wiping the shortcuts list for that device).

"effacer d'abord les raccourcis du périphérique" ? 
A bit long.

Reply:
A bit long but clear to me. As this is a label on a dialog it seems ok even if a bit long.